x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/knowledge_base/knowledge_base_bulk_import.spec.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { ELSER_ON_ML_NODE_INFERENCE_ID } from '@kbn/observability-ai-assistant-plugin/common/preconfigured_inference_ids';
+import {
+  clearKnowledgeBase,
+  getKnowledgeBaseEntriesFromEs,
+  setupKnowledgeBase,
+  waitForKnowledgeBaseReady,
+} from '../utils/knowledge_base';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const es = getService('es');
+  const observabilityAIAssistantAPIClient = getService('observabilityAIAssistantApi');
+
+  describe('Knowledge base: Bulk import operation with real ELSER model', function () {
+    // This test is intentionally skipped in both serverless and stateful
+    // since it is not meant to be run on CI but only as a manual test
+    before(async () => {
+      await setupKb(getService);
```

Comment:
Might just skip the helper. 
```suggestion
      await setupKnowledgeBase(getService, ELSER_ON_ML_NODE_INFERENCE_ID);
      await waitForKnowledgeBaseReady(getService);
```